src/SelectableBox/SelectableBoxSet.jsx

```diff
@@ -35,6 +36,7 @@ const SelectableBoxSet = React.forwardRef(({
         `pgn__selectable_box-set--${columns || DEFAULT_COLUMNS_NUMBER}`,
         className,
       ),
+      ...props,
```

Comment:
IIRC, I believe there was also some discussion around ensuring the Paragon documentation website page(s) pertaining to `SelectableBox.Set` included either the `aria-label` or `aria-labelledby` attribute(s) in the docs site examples themselves so its more evident to other consumers that either of these attributes is recommended / required(?).

Reply:
I've added details in the PropTypes definition. Do we think this information should be more visible? Such as getting it's own section on the doc site page?